### DISCUSSION TITLE
New computer

### DISCUSSION THREAD

**Discussion by robinsonmeredeth** [codefori/vscode-ibmi#2404] [Q&A]
Hello, I will be issued a new computer.  I will need to reinstall VS Code.  I don't recall how to install it and I would like to export my configuration and import into the new install.  Where can I locate the instructions for this?  Thanks.

**Comment by worksofliam** [comment]
Check out this page on the VS Code docs: https://code.visualstudio.com/docs/editor/settings-sync

**Reply by robinsonmeredeth** [reply]
Great!  Thank you!
